Bukkit/src/main/resources/plugin.yml

```diff
@@ -420,3 +420,4 @@ permissions:
       plots.done: true
       plots.continue: true
       plots.middle: true
+folia-supported: true
```

Comment:
Format, please put this in a higher location.

Reply:
I will change that